tvb/simulator/plot/tools.py

```diff
@@ -250,10 +259,164 @@ def plot_local_connectivity(cortex, cutoff=None):
     pyplot.legend(("Theoretical", "Typical", "Worst", "Best", "Cutoff"))
 
     # set the linewidth of the first legend object
-    #leg.legendHandles[0].set_linewidth(6.0)
-    #leg.legendHandles[1].set_linewidth(6.0)
-    #leg.legendHandles[2].set_linewidth(6.0)
-    #leg.legendHandles[3].set_linewidth(6.0)
+    # leg.legendHandles[0].set_linewidth(6.0)
+    # leg.legendHandles[1].set_linewidth(6.0)
+    # leg.legendHandles[2].set_linewidth(6.0)
+    # leg.legendHandles[3].set_linewidth(6.0)
+
+def temp_avg_timeseries(TAVG, EEG, sim, out, tt):
+    """
+    Temporal Averaged time-series
+    :param TAVG: numpy array
+    :param EEG: numpy array
+    :param sim: Simulator` instance
+    :param out: run function to perform the simulation
+    :param tt: numpy array
+    :return: output: 2D time Series
+    """
+
+    # Set up a Connectivity and its attributes.
+    conn = connectivity.Connectivity(load_default=True)
+    conn_coupling = coupling.Linear(a=0.042)
+
+    # Set up a model.
+    mod = models.Generic2dOscillator(a=-0.5, b=-15.0, c=0.0, d=0.02)
+
+    # Choose an integration scheme (noise or not?).
+    hiss = noise.Additive(nsig=numpy.array([0.015]))
+    heunint = integrators.HeunStochastic(dt=2 ** -6, noise=hiss)
+
+    # Build a Stimulus
+    conn.configure()
+    nodes = [35, 36]
+    stim_weights = numpy.zeros((conn.number_of_regions, 1))
+    stim_weights[nodes] = numpy.array([3.5, 0.0])[:, numpy.newaxis]
+    eqn_t = equations.PulseTrain()
+    eqn_t.parameters["onset"] = 500.0  # ms
+    eqn_t.parameters["tau"] = 5.0  # ms
+    eqn_t.parameters["T"] = 500.  # 0.002kHz repetition frequency
+
+    stimulus = patterns.StimuliRegion(temporal=eqn_t,
+                                      connectivity=conn,
+                                      weight=stim_weights)
+
+    # Record the ouput as:
+    # * Temporal Averaged time-series. By default only the first state
+    #       variable is recorded for the `Generic2dOscillator` model.
+    # * EEG
+    pr = projections.ProjectionSurfaceEEG(load_default=True)
+    ss = sensors.SensorsEEG.from_file(source_file="eeg_brainstorm_65.txt")
+    rm = region_mapping.RegionMapping(load_default=True)
+
+    rec = (monitors.TemporalAverage(period=1e3 / 2048.),
+           monitors.EEG(projection=pr, sensors=ss, region_mapping=rm, period=1e3 / 2048.))
+
+    figure()
```

Comment:
Only the lines related to figure should be placed here

Reply:
fixed